### PR TITLE
FEATURE: Linux compatibility

### PR DESCRIPTION
**Pull request by fnkr**

### CODE REVIEW

File: templates/include/letsencrypt.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+location ^~ /.well-known/acme-challenge/ {
+    alias {{ dehydrated.config.WELLKNOWN }}/;
```
Comment (Corvan) [inline templates/include/letsencrypt.conf.j2:2]: Should we add a remark (other than in meta) That this role depends on the dehydrated role? Documentation?

**Review by Corvan — APPROVED** [review]

File: templates/include/letsencrypt.conf.j2 (hunk 1 above)
Comment (fnkr) [reply templates/include/letsencrypt.conf.j2:2]: Role dependencies are managed in `meta/main.yaml`.

Comment (Corvan) [reply templates/include/letsencrypt.conf.j2:2]: Sure, but if someone chooses to use this role, they will not necessarily be notified about the dependency. Maybe documentation is not the right place but maybe e.g. one of GitHub's mechanisms to track dependencies?

Comment (Corvan) [reply templates/include/letsencrypt.conf.j2:2]: So that it becomes clear on which exact role-project this one depends

Comment (fnkr) [reply templates/include/letsencrypt.conf.j2:2]: For new projects the [proserver-ansible-template](https://github.com/punktDe/proserver-ansible-template) repository should be used to get started. The repository contains all necessary submodules and documentation. We could also add a README to this project (some roles have this, e.g. the [PostgreSQL role](https://github.com/punktDe/proserver-ansible-postgresql)).

**Pull request merged**